tests/test_xsamtools.py

```diff
@@ -48,6 +48,25 @@ def test_combine(self):
                 output = "test_bcftools_combined.vcf.gz"
                 vcf.combine(inputs, output)
 
+    def test_cli_vcf_merge(self):
+        inputs = ["tests/fixtures/a.vcf.gz", "tests/fixtures/b.vcf.gz"]
+        output = "test_bcftools_combined.vcf.gz"
+        # Test with some arguments that get passed on to bcftools
+        vcf.combine(inputs, output, '--force-samples', '--print-header')
```

Comment:
Can this check that the expected output appeared and possibly check that the file is the expected size (and not, say, zero bytes)?